portfolio/src/main/webapp/passions.html

```diff
@@ -40,14 +38,14 @@
         <div class="row h-100 align-items-center">
           <div class="col-12 text-center">
             <h1 class="font-weight-light" id="lighter-font">My Passions</h1>
-            <a class="btn btn-primary btn-xl text-uppercase js-scroll-trigger" href="#about">Find out more</a>
+            <a class="btn btn-primary btn-xl text-uppercase js-scroll-trigger" href="#passion-list">Find out more</a>
           </div>
         </div>
       </div>
     </div>
 
-    <!-- About Section -->
-    <section class="page-section" id="about">
+    <!-- Passion-List Section -->
```

Comment:
Minor note: these comments don't hurt, but since you already named the section very clearly `<section id="about">`, I find that your HTML code already speaks for itself.